keras_cv/metrics/coco/utils.py

```diff
@@ -96,3 +96,75 @@ def sort_bboxes(boxes, axis=5):
         )
 
     return boxes_sorted_list.stack()
+
+
+def match_boxes(y_true, y_pred, threshold, ious):
+    """matches bounding boxes from y_true to boxes in y_pred.
+
+    Args:
+        y_true: bounding box tensor of shape [num_boxes, 4+].
```

Comment:
I guess `num_boxes` could be different between y_true and y_pred. We might want to make it clear for that.